javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -7,25 +7,31 @@ import scalanative.runtime.unwind
 import scala.scalanative.meta.LinktimeInfo
 // TODO: Replace with j.u.c.ConcurrentHashMap when implemented to remove scalalib dependency
 import scala.collection.concurrent.TrieMap
+import scala.scalanative.runtime.Backtrace
 
 private[lang] object StackTrace {
   private val cache = TrieMap.empty[CUnsignedLong, StackTraceElement]
 
   private def makeStackTraceElement(
-      cursor: Ptr[scala.Byte]
+      cursor: Ptr[scala.Byte],
+      ip: CUnsignedLong
   )(implicit zone: Zone): StackTraceElement = {
     val nameMax = 1024
     val name = alloc[CChar](nameMax.toUSize)
     val offset = stackalloc[scala.Long]()
 
     unwind.get_proc_name(cursor, name, nameMax.toUSize, offset)
 
+    val fileline: Option[(String, Int)] =
+      Backtrace.decodeFileline(ip.toLong)
```

Comment:
The problem is, if `Backtrace.decodeFileline` throws an exception (even if we catch it), the exception will call `fillInStackTrace` (when `Throwable` is instantiated) which leads to here and endless loop :\

I'm wondering is there a way to suppress stacktraces (`writableStackTraces = false`) inside this function 🤔

Reply:
> I'm wondering is there a way to suppress stacktraces (`writableStackTraces = false`) inside this function

Maybe with sort of thread local?

Reply:
Thanks! I end up doing like this https://github.com/scala-native/scala-native/pull/3343/commits/0e2c67a56ea02568604bb5b7b9f0e0ef857a3f4a :)